wiki/Gameplay/Spectate/en.md

```diff
@@ -0,0 +1,11 @@
+---
+Spectating
```

Comment:
are you trying to add a tag here? if so, it should be like this

```suggestion
tags:
  - spectating
```

Reply:
gotcha - that's what i was going for. Is there a reason for tagging this way here and another way in Tournament articles (like ive done)? or is the tournament way just outdated somehow?

Reply:
Can you elaborate it bit more, I'm not sure if I understood your question correctly.